libcodechecker/server/session_manager.py

```diff
@@ -0,0 +1,541 @@
+# -------------------------------------------------------------------------
```

Comment:
Did you modified this file or just changed the path? Why github doesn't recognize that the path was changed?

Reply:
The first commit split the file into a serverside and a clientside code. The second commit modified the serverside file.